common/ideas/00_basic_ideas.txt

```diff
@@ -196,159 +104,273 @@ religious_ideas = {
+		land_maintenance_modifier = -0.1
 	}
+	smithian_economics = {
+		production_efficiency = 0.1
+ 	}
 
 	ai_will_do = {
-		factor = 0.75
+		factor = 0.85
+		modifier = {
+			factor = 1.5
+			inflation = 5
+		}
+		modifier = {
+			factor = 1.5
+			inflation = 10
+		}
+		modifier = {
+			factor = 2
+			has_reform = steppe_horde
+		}
+		modifier = {
+			factor = 1.3
+			num_of_cities = 20
+		}
+		modifier = {
+			factor = 1.3
+			num_of_cities = 50
+		}
 	}
 }
 
+#COME BACK TO THIS
+expansion_ideas = {
+	category = ADM
+	important = yes
 
-diplomatic_ideas = {
-	category = DIP
 	bonus = {
-		reduced_stab_impacts = yes
+ 		min_autonomy_in_territories = -0.1
 	}
 
-	foreign_embassies = {
-		diplomats = 1
+	additional_colonists = {
+		colonists = 1
 	}
-	cabinet = {
-		diplomatic_upkeep = 1
+	additional_merchants = {
+		merchants = 1
 	}
-	grand_banquets = {
-		diplomats = 1
+	faster_colonists = {
+		global_colonial_growth = 20
 	}
-	benign_diplomats = {
-		improve_relation_modifier = 0.25
+	factories = {
+		rival_border_fort_maintenance = -0.5
+		center_of_trade_upgrade_cost = -0.2
 	}
-	experienced_diplomats  = {
-		diplomatic_reputation = 2
+	additional_diplomats = {
+		diplomatic_upkeep = 1
 	}
-	flexible_negotiation = {
-		province_warscore_cost = -0.2
+	general_colonization_law  = {
+		colonists = 1
+		colonist_placement_chance = 0.05
 	}
-	diplomatic_corps = {
-		dip_tech_cost_modifier = -0.1
+	competetive_merchants  = {
+		global_trade_power = 0.2
 	}
 
 	ai_will_do = {
-		factor = 0.67
-
+		factor = 0.48
 		modifier = {
-			factor = 2
-			is_emperor = yes
+			factor = 10
+			OR = {
+				has_idea_group = exploration_ideas
+				is_colonial_nation = yes
+			}
 		}
 		modifier = {
-			factor = 1.5
-			vassal = 2
+			factor = 5
+			OR = {
+				tag = CAS
+				tag = POR
+				tag = SPA
+			}
+		}
+		modifier = {
+			factor = 10
+			NOT = { has_dlc = "Third Rome" } # Has siberian frontier instead
+			tag = RUS
+			num_of_cities = 20
+			any_owned_province = {
+				has_empty_adjacent_province = yes
+			}
+		}
+		modifier = {
+			factor = 0
+			NOT = { has_idea_group = exploration_ideas }
+			NOT = { is_colonial_nation = yes }
+			NOT = {
+				any_owned_province = {
+					has_empty_adjacent_province = yes
+				}
+			}
+		}
+		modifier = {
+			factor = 0.1
+			NOT = {
+				technology_group = western
+				technology_group = eastern
+				technology_group = muslim
+			}
 		}
 	}
 }
 
-
-offensive_ideas = {
-	category = MIL
+# Might need to come back to this.
+administrative_ideas = {
+	category = ADM
 
 	bonus = {
-		recover_army_morale_speed = 0.05
+		governing_capacity_modifier = 0.33
 	}
 
-	bayonet_leaders = {
-		leader_land_shock = 1
+	organised_mercenaries = {
+		merc_maintenance_modifier = -0.15
+		mercenary_manpower = 0.25
 	}
-	national_conscripts = {
-		global_regiment_recruit_speed = -0.1
+	adaptability = {
+		core_creation = -0.25
 	}
-	superior_firepower = {
-		leader_land_fire = 1
+	improved travel = {
+		local_friendly_movement_speed = 0.2
```

Comment:
I beliece local_friendly_movement_speed only works on province score (kinda like scorched earth). For a country modifier, use movement_speed